### PR TITLE
WIP: Feature/deploy to production

### PR DESCRIPTION
**Pull request by guillaumebriday**
Do not merge right now.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -63,16 +63,11 @@
     }
   },
   "scripts": {
-    "auto-scripts": {
-      "cache:clear": "symfony-cmd",
-      "assets:install --symlink --relative %PUBLIC_DIR%": "symfony-cmd",
-      "security-checker security:check": "script"
-    },
     "post-install-cmd": [
-      "@auto-scripts"
```
Comment (pifaace) [inline composer.json]: why did you remove 

> @auto-scripts

File: docker/php/php.ini [hunk 2]
```diff
@@ -1,8 +0,0 @@
-session.auto_start = Off
-date.timezone = Europe/Paris
-php_zip.dll
-
-# http://symfony.com/doc/current/performance.html
-opcache.max_accelerated_files = 20000
-realpath_cache_size = 4096K
```
Comment (pifaace) [inline docker/php/php.ini]: I think you don't need to remove all the file but just this line

File: roles/app/templates/nginx.conf.j2 [hunk 3]
```diff
@@ -0,0 +1,22 @@
+server {
+  listen 80;
+
+  index index.php index.html index.htm;
+  root /application/public; # default Laravel's entry point for all requests
```
Comment (pifaace) [inline roles/app/templates/nginx.conf.j2]: wrong c/p

File: docker/php/php.ini (hunk 2 above)
Comment (guillaumebriday) [reply docker/php/php.ini]: Ok I'll try tonight

File: composer.json (hunk 1 above)
Comment (guillaumebriday) [reply composer.json]: Because I had so many errors with this scripts. And I don't need it in production
